package.json

```diff
@@ -19,7 +19,7 @@
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v3.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-fix-api-error-messages",
```

Comment:
Note to self, update this after https://github.com/bravetechnologycoop/brave-alert-lib/pull/14 is merged.

Reply:
Done.